website/docs/docs/develop/getting-started/getting-started.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Getting started with dbt Cloud
-id: getting-started
+id: getting-started-dbt-cloud
```

Comment:
should the _redirects and sidebar.js files be updated?

Reply:
Yea just saw failing tests after I commit that suggestion 😵‍💫